build.gradle.kts

```diff
@@ -13,16 +14,19 @@ configurations.all {
         eachDependency {
             when (requested.module.toString()) {
                 "commons-codec:commons-codec" -> useVersion("1.10")
+                "org.jetbrains:annotations" -> useVersion("16.0.3")
             }
             when (requested.group) {
                 "org.jetbrains.kotlin" -> useVersion(kotlinVersion)
+                "org.seleniumhq.selenium" -> useVersion(seleniumVersion)
+                "org.slf4j" -> useVersion("1.7.25")
             }
         }
     }
 }
 
 dependencies {
-    api("com.atlassian.performance.tools:jira-actions:[2.1.0,4.0.0)")
+    api("com.atlassian.performance.tools:jira-actions:[3.10.2,4.0.0)")
```

Comment:
Needed because of stability, not APIs.

Reply:
You can leave it as is: `[2.1.0,4.0.0)`.
To update the locks, you can update only the one you need, plus the resulting conflicts:
```
./gradlew :dep --update-locks "com.atlassian.performance.tools:jira-actions,com.squareup.okio:okio,com.squareup.okhttp3:okhttp,com.google.guava:guava,net.bytebuddy:byte-buddy"
```
This way, the POM from `./gradlew genPom` is left intact:
```xml
    <dependency>
      <groupId>com.atlassian.performance.tools</groupId>
      <artifactId>jira-actions</artifactId>
      <version>[2.1.0,4.0.0)</version>
      <scope>compile</scope>
    </dependency>
```

- [ ] don't break major POM compatibility